### DISCUSSION TITLE
dropdown popup custom condition for # of items

### DISCUSSION THREAD

**Discussion by azim-b** [h2oai/wave#1214] [Ideas] [answered]
`popup` option for dropdown is a good addition. WDYT about customizing the threshold to trigger a popup? the default value is 100 items, and an option to change this would make it more flexible.

**Comment by mturoci** [comment, marked as answer]
In general we try to keep the component/card API concise and avoid bloat. The current threshold is UX improvement for people blindly pushing a lot of data to a single dropdown.

Not sure when the proposed attribute would be useful. Users can now manually control this behavior themselves thus implement their own python side threshold of needed.
